lib/team_stats.rb

```diff
@@ -1,5 +1,64 @@
 require_relative "stats"
-
 class TeamStats < Stats
 
+ def team_info(team_id)
+   team = team_by_id(team_id)
+    team_data = team.instance_variables.map { |key,value| ["#{key}".delete("@"), value = team.send("#{key}".delete("@").to_sym)]}.to_h
```

Comment:
Returns an array of the team object's instance variables, then iterates over that array, deletes the '@' from the front of the instance variable and assigns that as a key, then sets the value equal to the key by again removing the '@' and then passing that as a method call then returning the key, value pairs as a hash.